test/test_web.py

```diff
@@ -1958,6 +1959,38 @@ def test_get_project_json(self):
         assert 'n_completed_tasks' in data.keys(), list(data.keys())
         assert 'n_blogposts' in data.keys(), list(data.keys())
 
+    @with_context
+    def test_project_incomplete_notification(self):
+        """Test WEB JSON get project with project incomplete notification"""
+        admin = UserFactory.create()
+        admin.user_pref = {'language': ['en']}
+        project = ProjectFactory.create(owner=admin, info={'sched': 'default'})
+        task_user_pref = dict(languages=['sp'])
+        TaskFactory.create(project=project, user_pref=task_user_pref)
+        self.signin_user(admin)
+        url = '/project/%s/' % project.short_name
+
+        res = self.app_get_json(url)
+        data = json.loads(res.data)
+        assert 'notifications' not in data.keys(), list(data.keys())
+
+        project.info['sched'] = Schedulers.task_queue
+        res = self.app_get_json(url)
+        data = json.loads(res.data)
+        assert 'notifications' in data.keys(), list(data.keys())
```

Comment:
```
# Task Queue scheduler: verify notification message displays.
```

Also, is this updating the project repo and saving the new task queue preference via `self.app_get_json(url)`?

Reply:
`self.app_get_json(url)` is a test helper which only fetches data. The update to project scheduler is done via assignment in `project.info['sched'] = Schedulers.task_queue`